checks/cisco_temperature

```diff
@@ -238,9 +244,11 @@ def parse_cisco_temperature(info):
 
 
 def inventory_cisco_temperature(parsed):
+    discovery_options = host_extra_conf_merged(host_name(), discovery_cisco_dom_rules)
```

Comment:
We are determined to not add new features to plugins programmed against the old API.
In partiular, discovery parameters are now implemented similar to check parameters. See 5) of [this commit](https://github.com/tribe29/checkmk/commit/b428616c0e048bc08507dd1e96217a5f14bcacb9).